src/routes/controllers/ordersController/recentOrder.js

```diff
@@ -0,0 +1,16 @@
+import OrderModel from '../../../models/order';
+
+const recentOrder = async (req, res) => {
```

Comment:
rename to getRecentOrders for consistency

Reply:
all right